staubli_rx160_support/urdf/rx160l_macro.xacro

```diff
@@ -10,7 +18,7 @@
         <geometry>
           <mesh filename="package://staubli_rx160_support/meshes/rx160/visual/base_link.stl"/>
         </geometry>
-        <xacro:material_staubli_yellow />
+        <xacro:material_staubli_yellow_b />
```

Comment:
I know you've mentioned that your robot is actually this colour (RAL1018), I'd prefer all robots in the support pkgs to have their 'standard colours' (so RAL1028). Users with custom colours should then update a personal fork.

I understand this will mean more work for you, but if we start adding robots to support pkgs based on whatever paint job users happen to have, that will not work for the majority of our users.